ledger/test-evidence/src/main/scala/com/daml/test/evidence/generator/Main.scala

```diff
@@ -49,54 +33,19 @@ object Main {
     println(s"Wrote to $path")
   }
 
-  private def collectTestEvidence[TT: ClassTag, TS: ClassTag, TE](
-      scalaTestSuites: List[Suite],
-      ledgerApiSuites: List[LedgerTestSuite],
-      testEntry: (String, String, TT, Boolean, Option[TS]) => TE,
-  ): List[TE] =
-    List.empty
-      .concat(ScalaTestGeneratorSupport.testEntries(scalaTestSuites, testEntry))
-      .concat(LedgerApiTestGeneratorSupport.testEntries(ledgerApiSuites, testEntry))
-
   def main(args: Array[String]): Unit = {
-    val runpathList: List[String] = loadClasspath()
-      .map(_.split(":").toList)
-      .getOrElse(List.empty)
-
-    val ledgerApiTests = List()
-      .concat(suites.v1_14.default(timeoutScaleFactor = 0L))
-      .concat(suites.v1_14.optional(tlsConfig = None))
-
-    val testSuites: List[Suite] = ScalaTestAdapter.loadTestSuites(runpathList)
-
-    println("Writing security tests inventory..")
-
-    val securityTestEntries =
-      collectTestEvidence[SecurityTest, SecurityTestSuite, SecurityTestEntry](
-        testSuites,
-        ledgerApiTests,
-        SecurityTestEntry,
-      )
-
-    val reliabilityTestEntries =
-      collectTestEvidence[ReliabilityTest, ReliabilityTestSuite, ReliabilityTestEntry](
```

Comment:
You removed writing the reliability test evidence. Was that intentional? Are there no tests yet annotated with that in the daml repo?

Reply:
Yes, we do not have currently any reliability tests, that is why I removed it not to have empty files